hdfs-scheduler/src/main/java/org/apache/mesos/hdfs/scheduler/HdfsSchedulerModule.java

```diff
@@ -0,0 +1,16 @@
+package org.apache.mesos.hdfs.scheduler;
```

Comment:
I don't think we need this complex method of doing inheritance. We can simply use the constructor directly. I would like to keep our code as straight forward as possible.